src/utils/holding.ts

```diff
@@ -18,5 +18,7 @@ export const convertedValue = (holding: Holding) => {
   const globalCurrencyValue: number =
     euroValue * (globalCurrency.euro_rate || 1)
 
+  db.get('currencies').find({ code: holding.currency }).assign({convertedValue: globalCurrencyValue}).write()
```

Comment:
Just to confirm @alvesjtiago, do you mean removing this line? If so, for sure, I'll just import the helper function in `holding.ts` and call that in `export.ts`

Reply:
Exactly! Thank you

Reply:
I just pushed the changes, thanks!

Reply:
Thanks @clarencejychan, taking a look now!